ethereum/handler/contracts.go

```diff
@@ -12,16 +12,16 @@ import (
 	"go.vocdoni.io/dvote/log"
 )
 
-// EthereumContractNames is the list of supported smart contracts names
-var EthereumContractNames []string = []string{
-	"processes",
-	"namespaces",
-	"erc20",
-	"genesis",
-	"results",
-	"entities",
-	"ensRegistry",
-}
+const (
+	ENSregistryContractName       = "ensRegistry"
```

Comment:
This approach makes it hard to find for a developer, I'd rather use a common prefix, such as
`ContractName....`

Reply:
done :) thank you for the review